task/buildah-min/0.4/buildah-min.yaml

```diff
@@ -638,7 +636,7 @@ spec:
       echo "Running syft on the source directory"
       syft dir:"$(workspaces.source.path)/$SOURCE_CODE_DIR/$CONTEXT" --output "$syft_sbom_type"="$(workspaces.source.path)/sbom-source.json"
       echo "Running syft on the image filesystem"
-      syft dir:"$(cat /shared/container_path)" --output "$syft_sbom_type"="$(workspaces.source.path)/sbom-image.json"
+      syft oci-archive:"$(cat /shared/container_path)" --output "$syft_sbom_type"="$(workspaces.source.path)/sbom-image.json"
```

Comment:
I just realized that `syft` can probably read the image directly from the buildah storage if we just do `syft 
image:$IMAGE"`

Images can be gigantic, so if we could avoid pushing the image into the `/shared` volume that would be great

Reply:
For my proposed content-sets change, will we still hit this same performance hit?

Reply:
It will re-introduce the performance hit of starting a container and mounting its filesystem that this PR is removing. But it shouldn't be worse than the current state of the buildah task (prior to this PR), and supposedly we will one day be able to get rid of the ICM generation altogether

Reply:
Ideally, I think all we have should be the $IMAGE

> Would it be any better if we add the ICM generation script into the buildah-task image to run it in the same step where we build the container?

Maybe... :bulb: 
We could then, instead of using `buildah commit`, modify the Containerfile before building and add something like:

```Dockerfile
COPY content-sets.json /usr/share/buildinfo/content-sets.json
```

And because we're building with `--layers=false` (the default), that means the layers should get squashed the same way that `podman build --squash` would do it